tests/run-macros/tasty-tree-map/quoted_2.scala

```diff
@@ -42,7 +42,7 @@ object Test {
     println(identityMaped({ val x: Int | Any = 60; x }))
     println(identityMaped({ def f(x: => Int): Int = x; f(61) }))
     println(identityMaped({ type T[X] = X; val x: T[Int] = 62; x }))
-    println(identityMaped({ type T[X] = X match { case Int => String; case String => Int }; val x: T[String] = 63; x }))
+    println(identityMaped({ type T[X] = X match { case Int => String case String => Int }; val x: T[String] = 63; x }))
```

Comment:
What is the current state of matches without semicolons? We should have neg/pos tests with at least the folowing cases
```
X match { case Int => String; case String => Int }
X match { case Int => String case String => Int }
x match { case 1 => 3; case 2 => 4 }
x match { case 1 => 3 case 2 => 4 }
```